### PR TITLE
Update to NServiceBus.Azure.Transports.WindowsAzureServiceBus 7.1.0

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by Scooletz** [comment]
@Particular/azure-maintainers this is ready to move to doco maintainers. Unwip it please.

File: nservicebus/azure-service-bus/brokered-message-creation.md [hunk 1]
```diff
@@ -61,6 +62,6 @@ If the message content is in an unsupported or proprietary format, then the appl
 
 For a native message to be processed, NServiceBus endpoints using the Azure Service Bus transport require the following headers. These headers need to be stored as `BrokeredMessage` properties.
 
-partial: headers
+partial: header
```
Comment (SeanFeldman) [inline nservicebus/azure-service-bus/brokered-message-creation.md]: Why is it a singular and not plural?

**Comment by SeanFeldman** [comment]
Needs to be moved off RC0001 to a release version on MyGet (when available).

**Comment by SeanFeldman** [comment]
@Particular/docs-maintainers please review

File: samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config [hunk 2]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config [hunk 3]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net461" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net461" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config]: Here as well

File: samples/azure/custom-partitioning-asb/ASB_7/Subscriber1/packages.config [hunk 4]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/custom-partitioning-asb/ASB_7/Subscriber1/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/custom-partitioning-asb/ASB_7/Subscriber1/packages.config [hunk 5]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net461" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net461" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/custom-partitioning-asb/ASB_7/Subscriber1/packages.config]: here as well

File: samples/azure/native-integration-asb/ASB_7/MessageGenerator/packages.config [hunk 6]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/native-integration-asb/ASB_7/MessageGenerator/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/native-integration-asb/ASB_7/MessageGenerator/packages.config [hunk 7]
```diff
@@ -1,8 +1,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net461" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net461" />
   <package id="NUnit" version="3.5.0" targetFramework="net461" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/native-integration-asb/ASB_7/MessageGenerator/packages.config]: Here as well

File: samples/azure/native-integration-asb/ASB_7/Receiver/packages.config [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/native-integration-asb/ASB_7/Receiver/packages.config]: Package downgrade

File: samples/azure/native-integration-asb/ASB_7/Shared/packages.config [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/native-integration-asb/ASB_7/Shared/packages.config]: Package downgrade

File: samples/azure/performance-tuning-asb/ASB_7/FastAtomicReceiver/packages.config [hunk 10]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastAtomicReceiver/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/performance-tuning-asb/ASB_7/FastAtomicReceiver/packages.config [hunk 11]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastAtomicReceiver/packages.config]: Package downgrade

File: samples/azure/performance-tuning-asb/ASB_7/FastAtomicSenderReceiver/packages.config [hunk 12]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastAtomicSenderReceiver/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/performance-tuning-asb/ASB_7/FastAtomicSenderReceiver/packages.config [hunk 13]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastAtomicSenderReceiver/packages.config]: Here as well

File: samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicReceiver/packages.config [hunk 14]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicReceiver/packages.config]: Again

File: samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicReceiver/packages.config [hunk 15]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicReceiver/packages.config]: Again

File: samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicSenderReceiver/packages.config [hunk 16]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicSenderReceiver/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicSenderReceiver/packages.config [hunk 17]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastNonAtomicSenderReceiver/packages.config]: here as well

File: samples/azure/performance-tuning-asb/ASB_7/FastSender/packages.config [hunk 18]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastSender/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/performance-tuning-asb/ASB_7/FastSender/packages.config [hunk 19]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/FastSender/packages.config]: here as well

File: samples/azure/performance-tuning-asb/ASB_7/SlowAtomicReceiver/packages.config [hunk 20]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowAtomicReceiver/packages.config]: Did you pick the wrong side of comparison. Seems to indicate a version down grade?

File: samples/azure/performance-tuning-asb/ASB_7/SlowAtomicReceiver/packages.config [hunk 21]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowAtomicReceiver/packages.config]: here as well

File: samples/azure/performance-tuning-asb/ASB_7/SlowNonAtomicReceiver/packages.config [hunk 22]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowNonAtomicReceiver/packages.config]: Again

File: samples/azure/performance-tuning-asb/ASB_7/SlowNonAtomicReceiver/packages.config [hunk 23]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowNonAtomicReceiver/packages.config]: Again

File: samples/azure/performance-tuning-asb/ASB_7/SlowSender/packages.config [hunk 24]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowSender/packages.config]: Again

File: samples/azure/performance-tuning-asb/ASB_7/SlowSender/packages.config [hunk 25]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net452" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net452" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net452" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net452" />
```
Comment (danielmarbach) [inline samples/azure/performance-tuning-asb/ASB_7/SlowSender/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Publisher/packages.config [hunk 26]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Publisher/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Publisher/packages.config [hunk 27]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net461" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net461" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Publisher/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Shared/packages.config [hunk 28]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Shared/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Shared/packages.config [hunk 29]
```diff
@@ -1,6 +1,6 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Shared/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Subscriber/packages.config [hunk 30]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Subscriber/packages.config]: Again

File: samples/azure/polymorphic-events-asb/ASB_7/Subscriber/packages.config [hunk 31]
```diff
@@ -4,8 +4,8 @@
...
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.0.4" targetFramework="net461" />
+  <package id="NServiceBus" version="6.0.2" targetFramework="net461" />
+  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="7.1.0" targetFramework="net461" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.ServiceBus" version="3.4.3" targetFramework="net461" />
```
Comment (danielmarbach) [inline samples/azure/polymorphic-events-asb/ASB_7/Subscriber/packages.config]: Again

File: samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config (hunk 2 above)
Comment (SeanFeldman) [inline samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config]: Hmmm, not sure why this happened. We ( @boblangley  and I made sure to use the right "side").

**Comment by SeanFeldman** [comment]
@danielmarbach fixed the merge conflict issues.

@Particular/docs-maintainers please review. Note: had to apply a fix to `RawMessageSender_0.csproj`. It was using a wrong `HintPath`.

**Pull request merged**